server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+
+// TODO - Change this to use environment variables
+const port = 8081;
+
+const eventType = [ 
+    {label : "cop"},
+    {label : "landl"},
+    {label : "learningEvents"} 
+];
+
+// Dynamic endpoint receiving a param of eventtype to test
+app.get('/events/:eventtype', function (req, res) {
+    let foundEvent = eventType.find((event) => {
+        return event.label == req.params.eventtype;
+    });
+    if (foundEvent) {
+        res.send(getData(foundEvent));
+    } else {
+        res.sendStatus(404);
```

Comment:
I'm not sure if I would send a `404` if a specific event type doesn't exist, probably I'll just return an empty array

Reply:
This is just a suggestion but I thought if it doesn't find an event we could display a message saying no events instead of a 404 page

Reply:
Done, made it return an array instead if the eventtype is wrong.

Also this is for an API so there isn't much point in returning a nice message, an empty array is fine.